x/staking/keeper/validator_test.go

```diff
@@ -1059,6 +1059,12 @@ func TestUpdateValidatorCommission(t *testing.T) {
 	app, ctx, _, addrVals := bootstrapValidatorTest(t, 1000, 20)
 	ctx = ctx.WithBlockHeader(tmproto.Header{Time: time.Now().UTC()})
 
+	params := app.StakingKeeper.GetParams(ctx)
+
+	params.MinCommissionRate = sdk.NewDecWithPrec(5, 2)
```

Comment:
Can you generate this from a string, pretty hard to grok this

Reply:
```suggestion
	params.MinCommissionRate = sdk.MustNewDecFromStr("0.05")
```